sematic/api/endpoints/runs.py

```diff
@@ -15,6 +15,7 @@
 import flask
 import sqlalchemy
 from sqlalchemy.orm.exc import NoResultFound
+from sqlalchemy.sql.elements import BooleanClauseList
```

Comment:
Shouldn't this have been caught in its original PR? Does CI run `isort`?

Reply:
I don't think CI did, until a recent PR from Tudor